### DISCUSSION TITLE
Dev Problems in Github Codespaces

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#2040] [Q&A]
Trying to edit code in Github Codespaces online. I can build and run OK on port 3000, but if I do _pnpm dev_ then Codespaces will open both port 3000 & port 8080. The Flowise app displays briefly on the port 8080 browser tab then disappears leaving a blank browser tab.  Have tried setting up an .env file in package/ui to port 3001 but this doesn't help. There are no relevant errors in terminal.  I think this is related to Vite.

**Comment by HenryHengZJ** [comment]
mayb we need to set the vite port - https://stackoverflow.com/a/78193157

**Comment by jaredmcqueen** [comment]
even though a browser pops up using localhost:8080, use localhost:3000 instead
